packages/model/addon/-private/has-many.js

```diff
@@ -26,18 +26,18 @@ import { computedMacroWithOptionalParams } from './util';
 
   ```app/models/post.js
   import Model, { hasMany } from '@ember-data/model';
-
-  export default Model.extend({
-    comments: hasMany('comment')
-  });
+  
+  export default class PostModel extends Model {
+    @hasMany('comment') comments; 
+  }
   ```
 
   ```app/models/comment.js
   import Model, { belongsTo } from '@ember-data/model';
-
-  export default Model.extend({
-    post: belongsTo('post')
-  });
+  
+  export default class CommentModel extends Model {
+    @belongsTo('post') post; 
```

Comment:
I wonder how much we want to suggest this pattern

```
@belongsTo() post;
```

Reply:
I converted to decorators because that's what's being used in the Ember guides and other learning materials. I think we should be consistent. IMO, for someone just learning Ember its going to be confusing when they start looking at the API docs and they're doing something different then what they've learned elsewhere.

I'm assuming data's API are versioned the similar to the guides? So the changes would not be reflected in previous versions?  I know how the guides are built - no idea on the data API docs are built.